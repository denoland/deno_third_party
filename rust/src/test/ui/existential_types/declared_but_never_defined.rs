#![feature(existential_type)]

fn main() {}

// declared but never defined
existential type Bar: std::fmt::Debug; //~ ERROR could not find defining uses
