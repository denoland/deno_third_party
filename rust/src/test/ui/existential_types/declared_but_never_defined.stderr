error: could not find defining uses
  --> $DIR/declared_but_never_defined.rs:6:1
   |
LL | existential type Bar: std::fmt::Debug; //~ ERROR could not find defining uses
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

